config/module.config.php

```diff
@@ -118,13 +118,13 @@
             'orm_default' => [],
         ],
 
-        // SQL logger collector, used when ZendDeveloperTools and its toolbar are active
+        // SQL logger collector, used when LaminasDeveloperTools and its toolbar are active
```

Comment:
actually we are not using anywhere name in this format anymore - we have now laminas-developer-tools as the package name and `Laminas\DeveloperTools` as a module name (what is consistent with all other modules: `Laminas\Router`, `Laminas\Validator` etc)